src/Plugin/ImageEffect/CropEffectByCenter.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\crop\Plugin\ImageEffect\CropEffectByCenter.
+ */
+
+namespace Drupal\crop\Plugin\ImageEffect;
+
+use Drupal\Core\Image\ImageInterface;
+use Drupal\crop\Plugin\ImageEffect\CropEffect;
+
+/**
+ * Crops an image resource.
+ *
+ * @ImageEffect(
+ *   id = "crop_crop_center",
+ *   label = @Translation("Manual crop by center"),
+ *   description = @Translation("Applies manually provided crop by center to the image.")
+ * )
+ */
+class CropEffectByCenter extends CropEffect {
```

Comment:
Also, would be great to have test coverage. We have test for existing effect which can be used as an example.